checkout.html

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Adopt A Dog</title>
+  </head>
+  <body>
+    <header>
+      <figure class="company-logo">
+        <img src="images/adoptadog-logo.svg" alt="Company Logo">
+      </figure>
+
+      <figure class="cart-icon">
+        <img src="images/cart-icon.svg" alt="Cart Icon">
+        <figcaption>$123.56</figcaption>
+      </figure>
+      <button type="button" name="button">Checkout</button>
+
+      <nav>
+        <ul>
+          <li class="navlist"><a href="index.html">Home</a></li>
+          <li class="navlist"><a href="dogs.html">Dogs</a></li>
+          <li class="navlist"><a href="blog.html">Blog</a></li>
+        </ul>
+      </nav>
+
+    </header>
+    <main>
+
+      <h1>Checkout</h1>
+
+      <form class="checkout" action="checkout.html" method="get">
+
+        <label class="name">Name
```

Comment:
make sure to give each label and input the for attribute on the label and the id on the input